chainguard-dev/edu
# moving Events ref doc + removing enforce Admin bucket

## Type of change

This PR updates the GH action workflow to move the Events reference doc to the Administration bucket.

Thinking I'd feed two birds with one scone, this PR also gets rid of the Administration bucket in Enforce. 

### What should this PR do?

Resolves #1124 and #1113 

### Why are we making this change?

Moving the events ref doc will put it closer to our other events docs. Also, getting rid of the Administration sub bucket in the Enforce bucket will hopefully provide a clearer experience for visitors to the Academy. 

### What are the acceptance criteria? 



### How should this PR be tested?

Give everything a look and make sure it looks squared away.

FYI - Marking this as a draft PR until the template to the Enforce Events reference doc template kicks over ([merged here](https://github.com/chainguard-dev/mono/pull/15035))
